.github/workflows/hello_world_multiplatform.yaml

```diff
@@ -0,0 +1,78 @@
+name: Hello World (Multiplatform)
+
+on:
+  push:
+    branches:
+      - main
+      - v*-branch
+      - collab-*
+  pull_request:
+    branches:
+      - main
+      - v*-branch
+      - collab-*
+    paths:
+      - 'scripts/build/**'
+      - 'scripts/requirements*.txt'
+      - '.github/workflows/hello_world_multiplatform.yaml'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
At least for the main repo, we want to be using a specific version of runner so that we do not get any surprises when GitHub decides to change whatever the "latest" is.

Reply:
ok sounds good, updated to the current latest from https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories, just realized there's `macos-13` in bet there, would be "fun" to try it out later. :-)